### DISCUSSION TITLE
Showing growth time graph for last year as well

### DISCUSSION THREAD

**Discussion by anktrj** [monkeytypegame/monkeytype#2380] [Ideas] [answered]
As of now, the growth-time graph shows growth for only up to last month. 
It would great to see the graph for the last 3 months and last year as well.

**Comment by Miodec** [comment, marked as answer]
The graph is not limited to last month. Its limited to last 1000 results to reduce the load on the database. This could increase in the future.

**Reply by anktrj** [reply]
In that case, I have an architectural suggestion. I'll go through the code and suggest. Thanks for the answer btw.
